shell/platform/fuchsia/runtime/dart/utils/root_node_singleton.cc

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "root_node_singleton.h"
+
+namespace dart_utils {
+
+std::unique_ptr<sys::ComponentInspector> RootNodeSingleton::inspector_;
+std::mutex RootNodeSingleton::mutex_;
+
+void RootNodeSingleton::Initialize(sys::ComponentContext* context) {
```

Comment:
How do you feel about just calling this something like RootInspectNodeSingleton or RootInspectNode? The name node is a bit generic so we want to make it clear that this is for inspect.

Reply:
+10 for removing Singleton from the name, I like RootInspectNode

Reply:
Done.

Reply:
will remove the singleton -- just saw the comment after uploading.

Reply:
done.